### ISSUE TITLE
About face dataset and training procedures

### ISSUE THREAD

**Issue by SherryXTChen** [lyndonzheng/Pluralistic-Inpainting#65]
@lyndonzheng Thank you for the amazing work! I would like to experiment with training your model and there are several questions I would like to ask:
1. About the face dataset you mentioned with 24183 training images and 2824 test images, I cannot find that version of the dataset from the link you provided (http://mmlab.ie.cuhk.edu.hk/projects/CelebA.html). The aligned version of "In-The-Wild Images" folder has 202,599 images and CelebAMask-HQ Dataset (https://github.com/switchablenorms/CelebAMask-HQ) has 30000 images. Would you please direct me to the image folder you were using? 
2. I read your paper which stated that the training time using CelebA dataset and centered mask is roughly 3 days using 1 GPU with batch size 20. May I ask how many training iterations does it take since I don't have a GPU with the same specification which means training days may change on my GPU. 
3. Is the rest of the training procedure the same as the default parameter values in the options folder? More specfically, does the pretrained weight come from sizing images to 266 by 266 and crop them to 256 by 256 with random flip/rotation when training? 

Thank you for your time and consideration!

**Comment by lyndonzheng** [comment]
@SherryXTChen. 1. This is a duplicate of https://github.com/lyndonzheng/Pluralistic-Inpainting/issues/4, https://github.com/lyndonzheng/Pluralistic-Inpainting/issues/16, https://github.com/lyndonzheng/Pluralistic-Inpainting/issues/30.
2. We trained the model on CelebA-HQ with centered masks. The training iterations are around 200 epochs.  
3. Sure, the default parameters are suitable for centered masks.

**Comment by SherryXTChen** [comment]
@lyndonzheng Thank you for your response!
